### DISCUSSION TITLE
Csmoe citrus is English version for us guys?

### DISCUSSION THREAD

**Discussion by 1234man32** [community/community#34842] [Mobile]
Because I was just wondering is English of citrus csmoe because I couldn't figure out how to move through the menus on my own or to make it better and graphics or figure out the hud and if you guys give translation for English patch please that be cool

**Comment by HenSquared** [comment]
Hi @1234man32,

I'm not sure I understand the question. Is this question related to GitHub Mobile Apps? Or some other product?

Thanks!
